### PR TITLE
feat(iam): handle user role none

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/iam`
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #MANAGER-11393
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## New restricted App
This app loads when the user has no rights to display the manager
image

### CODE REVIEW

File: packages/manager/apps/restricted/src/public/translations/restricted/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "restricted_hello": "Hello FR !",
```
Comment (y4nnL) [inline packages/manager/apps/restricted/src/public/translations/restricted/Messages_fr_FR.json:2]: To remove

File: packages/manager/apps/restricted/src/public/translations/restricted/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "restricted_account_logout": "Me déconnecter",
+  "restricted_account_method_user": "Connecté en tant que sous-utilisateur",
+  "restricted_error_title": "Accès restreint",
+  "restricted_error_default": "Vous n’avez pas suffisamment de droit pour afficher cette page. Veuillez contacter votre administrateur",
+  "restricted_guide_card_a_description": "A very long long long long long long long long long long description of guide A",
+  "restricted_guide_card_a_title": "Guide A",
+  "restricted_guide_card_b_description": "Description of guide B",
+  "restricted_guide_card_b_title": "Guide B",
+  "restricted_guide_card_c_description": "Guide C has a typical description with some words",
+  "restricted_guide_card_c_title": "Guide C",
```
Comment (y4nnL) [inline packages/manager/apps/restricted/src/public/translations/restricted/Messages_fr_FR.json:14]: These translations will be changed as soon as we have the content

File: packages/manager/apps/restricted/src/restricted/LanguageMenu/language-menu.constants.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+export const SMALL_DEVICE_MAX_SIZE = '480px';
```
Comment (qpavy) [inline packages/manager/apps/restricted/src/restricted/LanguageMenu/language-menu.constants.ts:1]:
```suggestion
export const SMALL_DEVICE_MAX_SIZE = '30em';
```

File: packages/manager/apps/restricted/src/restricted/LanguageMenu/language-menu.styles.scss [hunk 4]
```diff
@@ -0,0 +1,6 @@
+.title {
+  color: #00185e;
+  font-weight: bold;
+  font-size: 14px;
```
Comment (qpavy) [inline packages/manager/apps/restricted/src/restricted/LanguageMenu/language-menu.styles.scss:4]:
```suggestion
  font-size: 0.875rem;
```

File: packages/manager/apps/restricted/src/restricted/restricted.styles.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  height: 2.75rem;
+}
+
+.restricted {
+  background-color: rgb(241, 249, 253);
+
+  .restricted-main {
+    box-sizing: border-box;
+    background-color: white;
+    max-width: 992px;
```
[the remark this answers is not in the document]
Comment (y4nnL) [reply packages/manager/apps/restricted/src/restricted/restricted.styles.scss:15]: Did you mean `62rem` ?

File: packages/manager/apps/restricted/src/restricted/restricted.styles.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
+.navbar {
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/restricted/src/restricted/restricted.styles.scss:1]: why not put this inside .restricted { .navbar } ?

**Review by qpavy — APPROVED** [review]

File: packages/manager/apps/restricted/src/index.tsx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    });
+
+  const root = createRoot(document.querySelector('#app'));
+
+  root.render(
+    <React.StrictMode>
+      <App />
+    </React.StrictMode>,
+  );
+});
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/restricted/src/index.tsx:83]: don't you need to catch a request error in your promise ?

**Review by Kranysys — APPROVED** [review]

File: packages/manager/apps/restricted/src/restricted/ErrorJumbotron/error-jumbotron.png
[the remark this answers is not in the document]
Comment (y4nnL) [reply packages/manager/apps/restricted/src/restricted/ErrorJumbotron/error-jumbotron.png:1]: A jumbotron is what it is, I don't get your point

**Review by aboungnaseng-ovhcloud — APPROVED** [review]

**Pull request merged**